catalog/redis-bucket/setters.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: setters
+data:
+  name: redis-bucket
+  namespace: config-control
+  network: default
+  project-id: $DEFAULT_PROJECT
```

Comment:
Assuming this is for BC. Currently rendered configs have `blueprints-project` for `project-id`.

Reply:
Correct, BC uses $DEFAULT_PROJECT as a magic placeholder. I'm hoping to have it replaced with a function: go/set-project-id.